### PR TITLE
Text fixes

### LINKED ISSUE #825: Align Text component with Figma typography
I just noticed we don't use figma tokens there, maybe something else has also changed?

### PR DESCRIPTION
**Pull request by rpominov**
## Description

1. New `Text` component, existing is deprecated. Closes #825
2. Deprecated `Paragraph` component (the replacement is `<Text as="p">`)
3. Added `textVariants` that can be used like this `css({ ...textVariants.regular })`
4. Removed `typography`, now `textVariants` or `textStyles` can be used instead
5. Added `theme.fontSize`, `theme.lineHeight`, `theme.fontWeight`, `theme.letterSpacing` that use values from Figma
6. Deprecated existing `theme.fontSize`, `theme.lineHeight`
7. Installed `Roboto Mono` font

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
- [ ] hi @TrySound, I need you to do
  - conceptual review (architecture, feature-correctness)
  
## Before requesting a review

- [x] made a self-review
- [x] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-designer/blob/main/apps/designer/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `designer/env-check.js` if mandatory

### CODE REVIEW

**Review by kof — APPROVED** [review]
great, we will migrate one by one and every time we touch a component for some other changes we should be migrating as well as a general rule

File: packages/design-system/src/components/text.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  },
+  defaultVariants: { variant: "regular" },
+});
+
+/**
+ * For use as a standalone, single-line text element. If you need a multiline element - use Paragraph.
+ */
+export const Text = styled("div", textStyles);
+
+export const Paragraph = styled("p", textStyles);
```
Comment (rpominov) [inline packages/design-system/src/components/text.ts:46]: @kof I just realised: the reason we had Paragraph before is probably `line-height`. Do we still need it, given it now has exactly same styles as Text?

Comment (rpominov) [reply packages/design-system/src/components/text.ts]: Removed `Paragraph`

**Review by TrySound — APPROVED** [review]

File: packages/design-system/src/components/text.ts (hunk 1 above)
Comment (rpominov) [reply packages/design-system/src/components/text.ts]: Going to merge to avoid collecting conflicts. Will do another PR if we need `Paragraph`

**Pull request merged**